client/resteasy-client/src/test/scala/com/cerner/beadledom/client/resteasy/ApacheHttpClient4Dot3EngineSpec.scala

```diff
@@ -5,15 +5,15 @@ import javax.ws.rs.NotFoundException
 import javax.ws.rs.client.{Client, Entity}
 
 import com.cerner.beadledom.client.TestResource
-import org.scalatest.{BeforeAndAfter, DoNotDiscover, FunSpec, ShouldMatchers}
+import org.scalatest.{BeforeAndAfter, DoNotDiscover, FunSpec, Matchers}
 import org.slf4j.{Logger, LoggerFactory}
 
 /**
   * @author John Leacox
   */
 @DoNotDiscover
 class ApacheHttpClient4Dot3EngineSpec(contextRoot: String, servicePort: Int)
```

Comment:
Is this still needed if you deleted the class?

Reply:
I wonder if we should just rename it to something like `DefaultHttpClientEngineSpec` since it still tests that the default `HttpClientEngine` used by `BeadledomResteasyClient` is working as expected.

Reply:
https://github.com/cerner/beadledom/commit/0284714eef35ecb4ca573e3e6c7dc3e92c148066